lib/reek/rake/task.rb

```diff
@@ -3,6 +3,7 @@
 require 'rake'
 require 'rake/tasklib'
 require 'pathname'
+require 'English'
```

Comment:
I'm afraid I'm just not 1337 enough for remembering the difference between $? and $!.;)